packages/sample-app/src/sample_decorators.ts

```diff
@@ -47,6 +47,16 @@ traceloop.withAssociationProperties(
     const completion = await sampleOpenAI.completion("TypeScript");
     console.log(completion);
 
-    await traceloop.reportScore({ chat_id: "789" }, 1);
+    await traceloop.getClient().userFeedback.create({
```

Comment:
I suggest (as mentioned in the python impl) that we'll split into two lines.. client retrieval and feedback creation.